### DISCUSSION TITLE
Bus::chain and the cloud task manager

### DISCUSSION THREAD

**Discussion by Kyon147** [stackkit/laravel-google-cloud-tasks-queue#97] [Q&A]
Hey @marickvantuil 

Thanks again for the awesome package, I have a Q on when using this driver and Job chaining.

When setting up multiple jobs and using the bus to run the chain, the drive pushed the entire payload for every job into one single task. Is this expected behaviour? 

I assume it is because we want the task to either run all successfully, or fail out?

**Comment by marickvantuil** [comment]
Thank you @Kyon147 

I'm not 100% sure what should be the correct behavior but I do know that the package does not change/modify or implement anything regarding chained jobs so I assume that what you described is indeed the intended/expected behavior, but we should check the native Laravel drivers to see what happens there (e.g. database or Redis driver)
